src/lib/forward.ts

```diff
@@ -95,8 +112,14 @@ export class EventForwarder {
 
     node.on("ready", (changedNode: ZWaveNode) => {
       // Dump full node state on ready event
-      const nodeState = dumpNode(changedNode);
-      notifyNode(changedNode, "ready", { nodeState });
+      this.clients.clients.forEach((client) =>
+        this.sendEvent(client, {
+          source: "node",
+          event: "ready",
+          nodeId: changedNode.nodeId,
+          node: dumpNode(changedNode, client.schemaVersion),
```

Comment:
So for backwards compat, this should be `nodeState`. That's the line that we're replacing.

Reply:
```suggestion
          nodeState: dumpNode(changedNode, client.schemaVersion),
```

Reply:
this makes sense to me